storyblok/storyblok-fieldtype
# Packages upgrade

This PR upgrades the packages used by the repo. I've not upgraded to the latest version eslint because one of the packages we use is not supporting that yet and it will throw an error when running `npm run serve`.

### How to test 
Try running all the commands available in the readme and make sure they work. Create a new custom field type and paste the code generated with `npm run build`. Make sure you set the name of the plugin on line 28 in `src/Plugin.vue` or the plugin won't work.
Try also [local development environment](https://www.storyblok.com/docs/plugins/enable-local-dev-mode), please.
